luaui/Include/rml_utilities/utils.lua

```diff
@@ -0,0 +1,119 @@
+-- RmlWidgets Utils
+-- Common utility functions for RmlUi widgets
+
+-- This is to ensure common class groups are loaded and providing this in the context manage is too early.
+local ccg = VFS.Include("luaui/Include/rml_utilities/common_class_groups.lua")
+
+-- Make utils global so all widgets share the same instance
+if not WG.rml_utils then
+    local utils = {}
+    WG.rml_utils = utils
+end
+local utils = WG.rml_utils
+
+-- Helper to combine multiple class strings
+function utils.combineClasses(...)
+    local args = {...}
+    local result = {}
+    
+    for _, classStr in ipairs(args) do
+        if classStr and classStr ~= "" then
+            table.insert(result, classStr)
+        end
+    end
+    
+    return table.concat(result, " ")
+end
+
+
+
+-- RML Widget initialization helper
+function utils.initializeRmlWidget(widget, initParams)
+    -- Validate required parameters
+    if not initParams.widgetId then
+        return false
+    end
+    if not initParams.modelName then
+        return false
+    end
+    if not initParams.rmlPath then
+        return false
+    end
```

Comment:
Maybe collect some feedback here before returning:

```suggestion
    local failures = {}
    for _, property in ipairs { "widgetId", "modelName", "rmlPath" } do
        if not initParams[property] then
            table.insert(failures, property)
        end
    end
    if next(failures) then
        local msg = "Could not initialize widget. Failed to validate: "
        Spring.Log("RML", LOG.WARNING, msg .. " " .. table.concat(failures, ", ")
        return false
    end
```